trees/binaryheap/binaryheap.go

```diff
@@ -13,14 +13,15 @@ package binaryheap
 
 import (
 	"fmt"
+	"strings"
+
+	"github.com/emirpasic/gods/containers"
 	"github.com/emirpasic/gods/lists/arraylist"
-	"github.com/emirpasic/gods/trees"
 	"github.com/emirpasic/gods/utils"
-	"strings"
 )
 
 func assertTreeImplementation() {
-	var _ trees.Tree = (*Heap)(nil)
+	var _ containers.Container = (*Heap)(nil)
```

Comment:
Yeah, I realized that towards the end, so I left another comment below in Tree interface.